components/ContributorCard.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+  <v-card flat class="ContriCard">
+    <v-avatar size="100px">
+      <img :src="iconPath" />
+    </v-avatar>
+    <span class="ContriText ContriName">{{ name }}</span>
+    <span class="ContriText ContriRole">{{ role }}</span>
+  </v-card>
+</template>
+
+<script>
+export default {
+  props: {
+    iconPath: {
+      type: String,
+      required: true
+    },
+    name: {
+      type: String,
+      required: true
+    },
+    role: {
+      type: String,
+      required: true
+    }
+  }
+}
+</script>
+
+<style lang="scss" scoped>
+.ContriCard {
+  display: inline-block;
+  width: 100px;
+  height: 145px;
+}
+.ContriCard img {
+  object-fit: cover;
+  border-radius: 50%;
+  border: 1px solid #f3f3f4;
+  box-sizing: border-box;
```

Comment:
```suggestion
  > img {
    object-fit: cover;
    border-radius: 50%;
    border: 1px solid #f3f3f4;
    box-sizing: border-box;
  }
```
この方がコンパクトに書けます。
正しく動くはずですが、確認していただけると助かります。

Reply:
コンポーネントのスタイルはimport先の全コンポーネントに影響するもんだと思ってました。正常に動作したのでcommitしました